WickedEngine/Utility/basis_universal/transcoder/basisu_containers.h

```diff
@@ -133,8 +133,8 @@ namespace basisu
       static inline void construct(T** p) { memset(p, 0, sizeof(T*)); }
       static inline void construct(T** p, T* init) { *p = init; }
       static inline void construct_array(T** p, size_t n) { memset(p, 0, sizeof(T*) * n); }
-      static inline void destruct(T** p) { p; }
-      static inline void destruct_array(T** p, size_t n) { p, n; }
+      static inline void destruct(T**) {}
```

Comment:
It triggers "unused variable" because it's included in our code. 
But it's not one of the warnings I decided to enable in the end, so I reverted the change.